src/logs/logService.ts

```diff
@@ -61,38 +61,30 @@ export class LogService {
   // TODO: readableStream cannot be used until updates are made in jsforce and sfdx-core
   @elapsedTime()
   public async getLogs(options: ApexLogGetOptions): Promise<LogResult[]> {
-    const logIdList = await this.getLogIds(options);
-    const logPaths: string[] = [];
-    const connectionRequests = logIdList.map(async (id) => {
-      const url = `${this.connection.tooling._baseUrl()}/sobjects/ApexLog/${id}/Body`;
-      const logRecord = await this.toolingRequest(url);
+    const logs = (
+      await Promise.all(
+        (await this.getLogIds(options)).map(async (id) => ({
+          ...(await this.getLogById(id)),
```

Comment:
we already had a method for doing this so it uses that instead of doing it manually